tests/chart_tests/test_auth_sidecar.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from tests.chart_tests.helm_template_generator import render_chart
+
+from .. import supported_k8s_versions
+from . import get_containers_by_name
+
+
+@pytest.mark.parametrize("kube_version", supported_k8s_versions)
+class TestAuthSidecar:
+    def test_auth_sidecar_config_defaults(self, kube_version):
+        """Test logging sidecar config with defaults"""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={},
+            show_only=[
+                "templates/dag-deploy/dag-server-auth-sidecar-configmap.yaml",
+                "templates/flower/flower-auth-sidecar-configmap.yaml",
+                "templates/webserver/webserver-auth-sidecar-configmap.yaml",
+            ],
+        )
+        assert len(docs) == 0
```

Comment:
We should re-use the show_only list since it is always the same:

```suggestion
@pytest.mark.parametrize("kube_version", supported_k8s_versions)
class TestAuthSidecar:
    show_only = [
        "templates/dag-deploy/dag-server-auth-sidecar-configmap.yaml",
        "templates/flower/flower-auth-sidecar-configmap.yaml",
        "templates/webserver/webserver-auth-sidecar-configmap.yaml",
    ]

    def test_auth_sidecar_config_defaults(self, kube_version):
        """Test logging sidecar config with defaults"""
        docs = render_chart(kube_version=kube_version, values={}, show_only=self.show_only)
        assert len(docs) == 0
```